public/app/core/components/NavBar/NavFeatureHighlight.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { css } from '@emotion/css';
+import { useStyles2 } from '@grafana/ui';
+import { GrafanaTheme2 } from '@grafana/data';
+
+export interface Props {
+  children: JSX.Element;
+  text: string;
+}
+
+export const NavFeatureHighlight = ({ children, text }: Props): JSX.Element => {
+  const styles = useStyles2(getIconStyles);
+  return (
+    <div className={styles.icon}>
+      {children}
+      <span className={styles.badge}>
+        {text} <i />
```

Comment:
nit: can we use something other than an `<i />` here? if it's purely decorative, can this be a `:before`/`:after` on the parent span instead?